packages/rich-text-editor/src/RichTextEditor/components/ToggleIconButton/ToggleIconButton.tsx

```diff
@@ -37,6 +37,7 @@ export const ToggleIconButton: React.VFC<ToggleIconButtonProps> =
           ref={ref}
           aria-pressed={isActive}
           aria-label={label}
+          onMouseDown={e => e.preventDefault()}
```

Comment:
@harish-everest this is out of scope for your work but any chance I could get you to add `type="button"` to this as well while you're here? 🙂